src/Compilers/CSharp/Portable/CSharpExtensions.cs

```diff
@@ -1236,7 +1236,7 @@ public static ISymbol GetDeclaredSymbol(this SemanticModel semanticModel, Variab
         /// <summary>
         /// Given a declaration pattern syntax, get the corresponding symbol.
```

Comment:
I'd rather not introduce an inconsistency with all the other places where this comment appears (e.g. for SingleVariableDesignationSyntax and VariableDeclaratorSyntax above)

---
In reply to: [80752496](https://github.com/dotnet/roslyn/pull/13999#discussion_r80752496) [](ancestors = 80752496)